src/Redux/books/books.js

```diff
@@ -1,30 +1,48 @@
 const ADD_BOOK = 'bookstore/books/ADD_BOOK';
 const REMOVE_BOOK = 'bookstore/books/REMOVE_BOOK';
 
-let initialCount = 0;
+let initialCount = 1;
 const initialState = {
-  books: [],
+  books: [
+    {
+      id: 1,
+      title: 'Rich Dad Poor Dad',
+      author: 'Robert Kiyosaki',
+    },
+    {
+      id: 2,
+      title: 'Why We Sleep',
+      author: 'Matthew Walker',
+    },
+    {
+      id: 3,
+      title: 'The richest man in babylon',
+      author: 'George S. Clason',
+    },
+  ],
 };
 /* eslint-disable no-plusplus */
-const addBook = (title) => ({
+const addBook = ({ title, author }) => ({
   type: ADD_BOOK,
-  id: ++initialCount,
+  id: initialCount++,
   title,
+  author,
 });
 
 const removeBook = (id) => ({
```

Comment:
After adding multiple books followed by deleting a few books, it does not delete the correct book that I want to delete. The problem may lay in the id assigned to the newly added books and the id of existing books in the array. Consider fixing it.